@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreGetWCAs.sqf

```diff
@@ -263,5 +263,8 @@ if (_heli getVariable "fza_ah64_rtrbrake") then {
 if (_fcrDamage >= SYS_FCR_DMG_THRESH) then {
     _wcas pushBack [WCA_ADVISORY, "FCR FAULT", "FCR FAULT"];
 };
+if (isTouchingGround _heli) then {
```

Comment:
It is related in that it's the indication that alerts the pilot to the fact that heading hold is enabled because the helicopter is no longer on the squat (weight on wheels) switch